Documentation/dynamic-rupture.rst

```diff
@@ -299,20 +299,44 @@ The TP parameters for which no spatial dependence has been implemented are defin
 
   &DynamicRupture
   thermalPress = 1                  ! Thermal pressurization 0: inactive; 1: active
-  IniTemp = 483.15                  ! Initial temperature [K]
-  IniPressure = -80.0e6             ! Initial pore pressure; have to be added to normal stress in your initial stress yaml file [Pa]
-  alpha_th = 1.0e-6                 ! Thermal diffusivity [m^2/s]
-  rho_c = 2.7e6                     ! Specific heat [Pa/K]
-  TP_lambda = 0.1e6                 ! Pore pressure change per unit temperature [Pa/K]
+  TP_IniTemp = 483.15                  ! Initial temperature [K]
+  TP_IniPressure = -80.0e6             ! Initial pore pressure; have to be added to normal stress in your initial stress yaml file [Pa]
+  TP_hydraulicDiffusivity = 1.0e-6     ! Thermal diffusivity [m^2/s]
+  TP_heatCapacity = 2.7e6              ! Specific heat [Pa/K]
+  TP_undraindedTPResponse = 0.1e6      ! Pore pressure change per unit temperature [Pa/K]
 
 Two additional thermal pressurization parameters are space-dependent and therefore have to be specified in the dynamic rupture yaml file:
 
 .. code-block:: YAML
 
   !ConstantMap
   map:
-    alpha_hy: 1e-4                  # Hydraulic diffusivity [m^2/s]
-    TP_half_width_shear_zone: 0.01  # Half width of shearing zone [m]
+    tp_hydraulic_diffusivity: 1e-4   # Hydraulic diffusivity [m^2/s]
+    tp_half_width_shear_zone: 0.01   # Half width of shearing zone [m]
 
 TP generates 2 additional on-fault outputs: Pore pressure and temperature (see fault output).
 
+Convert parameter files for new DR release
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the reimplementation of Dynamic Rupture in C++, we refactored some parameter names to make things more consistent.
```

Comment:
maybe: `While porting Dynamic Rupture to C++, we refactored a few parameter names...`
(add version number, or PR id, or commit)